src/commandHandlers/bio.ts

```diff
@@ -25,6 +25,15 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
         return embed;
     }
 
+    // Delete bio
+    if(field === 'delete' && !content && !mention) {
+        await db
+        .collection('users')
+        .doc(message.author.id)
+        .delete();
```

Comment:
Great idea 👍 - should this maybe delete all `bio` fields? `'description', 'github', 'twitter', 'youtube', 'instagram', 'linkedin', 'location', 'timezone'`

Reply:
It deletes all fields - Since it deletes the document. Are you wanting it to only delete the bio fields?

Reply:
Would you prefer something like this?
```Typescript
await db
  .collection('users')
  .doc(message.author.id)
  .update({
      bio: firebase.firestore.FieldValue.delete()
  });
```

Reply:
Ah yes, good point @AllanRegush 👍 

I would prefer just deleting the `bio`, but people might prefer to delete everything, so I think what you have is probably better 🤓